src/common/utxobased/engine/makeUtxoEngineState.ts

```diff
@@ -1213,7 +1121,7 @@ const processRawUtxo = async (
 
       // Legacy UTXOs need the previous transaction hex as the script
       // If we do not currently have it, add it to the queue to fetch it
-      tx = await processor.fetchTransaction(utxo.txid)
+      tx = (await processor.fetchTransactions({ txId: utxo.txid }))[0]
```

Comment:
`;[tx] = await processor.fetchTransactions({ txId: utxo.txid })` I think this will work.

Reply:
Done.